how-to/1_Read_and_visualise/NikonDataReader.ipynb

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
I'm not sure what the changes to the NikonDataReader are? If you merge master into your branch it should show what the difference is.